openlibrary/plugins/openlibrary/js/jquery-ui/index.js

```diff
@@ -0,0 +1,47 @@
+function init() {
+    return import(
+        /* webpackChunkName: "ui" */
+        './ui'
+    );
+}
+
+/**
+ * Returns a placeholder jQuery UI function which when invoked will
+ * load jQuery UI and make the element functional.
+ * @param {string} fnName to call when loaded e.g. sortable,tabs,colorbox
+ */
+export function placeholder(fnName) {
+    return function () {
+        // only if the selector matches load the additional code and wire it up.
+        if ($(this).length) {
+            init().then(() => {
+                if (fnName === 'colorbox') {
+                    // set option to open immediately since loading was delayed.
+                    arguments[0].open = true;
+                }
```

Comment:
Removed this in favor of loading jquery-ui as soon as `colorbox` is called for init instead of waiting for user interaction; this is still an improvement over master since the loading of jquery-ui is non-blocking.